frontend/src/app/app-core/components/header/header.component.ts

```diff
@@ -3,17 +3,28 @@ import {Store} from '@ngrx/store';
 import * as fromRoot from '../../reducers';
 import {Observable} from 'rxjs/Observable';
 import {getHeaderTitle} from '../../reducers/main-frame/main-frame.selectors';
+import {TranslationService} from "../../../../translation/translation.service";
 
 @Component({
   selector: 'app-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.css'],
 })
-export class HeaderComponent {
+export class HeaderComponent implements OnInit{
 
+  languages: string [];
   title$: Observable<string>;
 
-  constructor(private store: Store<fromRoot.AppState>) {
+  constructor(private store: Store<fromRoot.AppState>, private translationService: TranslationService) {
     this.title$ = this.store.select(getHeaderTitle);
   }
+
+  ngOnInit(){
+    this.languages = this.translationService.languages;
```

Comment:
getter setter pattern is a common pattern in developer world, not always important but you can avoid some stupid discussions if you follow that approach